packages/rosaenlg-lib/test/pug/fr_FR/sentence_negative.pug

```diff
@@ -202,6 +208,65 @@ mixin SUJET_MP_ref(obj, params)
     },
   };
 
+//-  Le tyran n'a vraiment fourni aucune aide au jeune aventurier.
+
+
+- const SUJET_TYRAN = {};
+mixin SUJET_TYRAN_ref(obj, params)
+  | le tyran
+  - setRefNumber(SUJET_TYRAN, 'S');
+  - setRefGender(SUJET_TYRAN, 'tyran');
+- SUJET_TYRAN.ref = SUJET_TYRAN_ref;
+
+mixin SUJET_TYRAN_refexpr(obj, params)
+  if params.deprel === 'subj'
+    | il
+  else
+    | cet homme
+- SUJET_TYRAN.refexpr = SUJET_TYRAN_refexpr;
+
+- const COD_AIDE = {};
+mixin COD_AIDE_ref(obj, params)
+  | aucune aide
+  - setRefGender(COD_AIDE, 'F');
+  - setRefNumber(COD_AIDE, 'S');
+- COD_AIDE.ref = COD_AIDE_ref;
+
+- const COI_AVENTURIER = {};
+mixin COI_AVENTURIER_ref(obj, params)
+  | le jeune aventurier
+  - setRefGender(COI_AVENTURIER, 'aventurier');
+  - setRefNumber(COI_AVENTURIER, 'S');
+- COI_AVENTURIER.ref = COI_AVENTURIER_ref;
+
+-
+  const sentenceTyran = {
+    subjectGroup: {
+      subject: SUJET_TYRAN,
+      params : { deprel: 'subj' }
+    },
+    negative: true,
+    negativeAdverb: '',
+    modifierAdverb : 'vraiment',
+    verbalGroup: {
+      verb: 'fournir',
+      tense: 'PRESENT',
+      aux:'AVOIR',
+    },
+    objGroups: [
+      {
+        type: 'DIRECT',
+        obj: COD_AIDE,
+        params : { REPRESENTANT: 'ref' }
```

Comment:
this is a good trick
can you please document it in `sentences.adoc`?

Reply:
done